rakefile-dotnet.rb

```diff
@@ -0,0 +1,373 @@
+require 'albacore'
+
+msbuild_command = "C:/Program Files (x86)/MSBuild/14.0/Bin/MSBuild.exe"
+if !File.file?(msbuild_command)
+  raise "MSBuild not found"
+end
+
+nuget_command   = ".nuget/nuget.exe"
+gitlink_command = "packages/gitlink.2.3.0/lib/net45/GitLink.exe"
+xunit_command   = "packages/xunit.runner.console.2.0.0/tools/xunit.console.exe"
+
+solution        = "FakeItEasy-NetCore.sln"

```

Comment:
@jeremymeng (or anyone), do you know why executing 

```C:/Program Files (x86)/MSBuild/14.0/Bin/MSBuild.exe" FakeItEasy-NetCore.sln /target:Build /p:configuration=Release /nr:false /verbosity:minimal /nologo /fl /flp:LogFile=artifacts/logs/Build.log;Verbosity=Detailed;PerformanceSummary /p:NuGetPackagesDirectory=C:/PortableApps/Documents/Source/GitHub/FakeItEasy/packages```

here results in 

```
  Project FakeItEasy.Tests (.NETCoreApp,Version=v1.0) will be compiled because expected inputs are missing
  Compiling FakeItEasy.Tests for .NETCoreApp,Version=v1.0
C:\PortableApps\Documents\Source\GitHub\FakeItEasy\tests\FakeItEasy.Tests.dotnet\project.json(44,35): error NU1001:
The dependency FluentAssertions >= 4.6.3 could not be resolved. [C:\PortableApps\Documents\Source\GitHub\FakeItEasy\tests\FakeItEasy.Tests.dotnet\FakeItEasy.Tests.dotnet.xproj]
C:\PortableApps\Documents\Source\GitHub\FakeItEasy\tests\FakeItEasy.Tests.dotnet\project.json(45,34): error NU1001:
The dependency Microsoft.CodeAnalysis.Analyzers >= 1.1.0 could not be resolved. [C:\PortableApps\Documents\Source\GitHub\FakeItEasy\tests\FakeItEasy.Tests.dotnet\FakeItEasy.Tests.dotnet.xproj]
C:\PortableApps\Documents\Source\GitHub\FakeItEasy\tests\FakeItEasy.Tests.dotnet\project.json(59,55): error NU1001:
The dependency Microsoft.CodeAnalysis.Analyzers >= 1.1.0 could not be resolved. ```

Can one note build ` .xproj` from within a .`sln` using MSBuild? The rakefile runs the `nuget restore` earlier, so I'd think that wasn't it, but 
```
dotnet restore
dotnet build -c Release
``` 

does pass. 

Is there any way forward but to drop the project from the `.sln`? Which I'm loathe to do.

Thank for any help you can bring.